src/components/feedbacks/FeedbackItem.vue

```diff
@@ -1,28 +1,24 @@
 <template>
   <BaseBox>
     <article>
-      <p>{{}}</p>
+      <p v-if="roadmap">{{ feedback.status }}</p>
       <!-- status - activated in the Roadmap, hidden in the FeedbackList -->
       <div class="feedback__votes">
         <ArrowUp />
-        <p>{{}}</p>
-        <!-- upvotes -->
+        <p>{{ feedback.upvotes }}</p>
       </div>
       <div class="feedback__content">
-        <h2>{{}}</h2>
-        <!-- title -->
-        <p>{{}}</p>
-        <!-- content -->
-        <BaseButton>{{}}</BaseButton>
-        <!-- category -->
+        <h2>{{ feedback.title }}</h2>
+        <p>{{ feedback.description }}</p>
+        <BaseButton>{{ feedback.category }}</BaseButton>
       </div>
       <div class="feedback__comments">
         <IconComments />
-        <p>{{}}</p>
-        <!-- comments.length move to script and create a correct prop or another feature - to check
-    possible usage - v-if - if there iss no comment the number should be in grey color -->
-        <!-- attach NoFeedback and make like in the BaseButton props with Boolean and if true attach feedback item if false - v-else - attach NoFeedback component
-    To consider - where hould be attached NoFeedback and v-if - in FeedBackItem or in FeedbackList -->
+        <p v-if="feedback.comments">
+          {{ feedback.comments.length }}
```

Comment:
feedback.comments.length || 0 and then you don't need v-if and else. Styles can be bind